app/views/handlers/feedback.py

```diff
@@ -64,7 +63,7 @@ def form(self) -> QuestionnaireForm:
             question_schema=self.question_schema,
             answer_store=self._questionnaire_store.answer_store,
             list_store=self._questionnaire_store.list_store,
-            metadata=self._questionnaire_store.metadata,
+            metadata=self._questionnaire_store.metadata,  # type: ignore
```

Comment:
This shouldn't be ignored. metadata on questionnaire should be typed as `Mapping[str, Any]`, then any typing of metadata such as the one in `generate_form` should be updated to match.

Reply:
I've changed it to `Mapping[str, Any]` and removed the ignores but it meant changing types in `questionnaire_form.py`, `submit_questionnaire.py`, `section_summary_context.py`, `globals.py` etc.